maps/snaxi.dm

```diff
@@ -26,7 +26,8 @@
 		)
 	enabled_jobs = list(/datum/job/trader)
 
-	event_blacklist = list(/datum/event/radiation_storm)
+	event_blacklist = list(/datum/event/radiation_storm,/datum/event/carp_migration,/datum/event/rogue_drone,/datum/event/immovable_rod,
+						/datum/event/meteor_wave,/datum/event/meteor_shower,/datum/event/thing_storm)
```

Comment:
What the hells a thingstorm

Reply:
It contains a variety of storms to throw at the station including gore, fireworks, blob, etc.